Form-Controls/index.html

```diff
@@ -1,28 +1,79 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>My form exercise</title>
-  <meta name="description" content="">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="styles.css">
-</head>
-<body>
-  <header>
-    <h1>Product Pick</h1>
-  </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>My form exercise</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=EB+Garamond&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header>
+      <h1>Order you T-shirt here!</h1>
+    </header>
+    <main>
+      <form>
+        <div class="form-container">
+          <form>
+            <label for="name">Name:</label>
+            <input
+              type="text"
+              id="name"
+              name="name"
+              required
+              pattern="[a-zA-Z ]{3,}"
+              title="Name must contain only letters and at least 3 characters"
+            />
 
-  </main>
-  <footer>
-    <!-- change to your name-->
-    <h2>By HOMEWORK SOLUTION</h2>
-  </footer>
-</body>
+            <label for="email">Email:</label>
+            <input
+              type="email"
+              id="email"
+              name="email"
+              required
+              title="Email must be in valid format (example@domain.com)"
+            />
 
-</html>
\ No newline at end of file
+            <label for="color">Choose the Color:</label>
+            <select id="color" name="color" required>
+              <option value="" disabled selected>Choose a color</option>
+              <option value="black">Black</option>
+              <option value="white">White</option>
+              <option value="beinge">Beinge</option>
```

Comment:
```suggestion
              <option value="beige">Beige</option>
```
You can, and should, install a spellchecker in VSCode.